extensions/control-plane/api/management-api/management-api-test-fixtures/src/testFixtures/java/org/eclipse/edc/connector/controlplane/test/system/utils/Participant.java

```diff
@@ -58,6 +58,8 @@
  */
 public class Participant {
 
+    public static final String CONSUMER = "consumer";
+    public static final String PROVIDER = "provider";
```

Comment:
the `Participant` class is not supposed to know about the ids that are used by the tests, these constants shouldn't stay here.